Examples/Run/Reconstruction/Common/RecCKFTracks.cpp

```diff
@@ -34,11 +37,21 @@
 
 #include <boost/filesystem.hpp>
 
+#include "RecInput.hpp"
```

Comment:
Full include path?

Reply:
The *.cpp files are in the same folder. I'm following the approach for those files used for executables in `Examples/Run`.

Reply:
Hm, alright.